src/main/java/sirius/db/mixing/FieldLookupCache.java

```diff
@@ -47,16 +46,16 @@
      * @param id    the id of the entity to resolve
      * @param field the field to resolve
      * @param <E>   the generic type of the entitiy
-     * @return the value of the field or an empty value if either the field is empty or the given ID was <tt>null</tt>
+     * @return the value of the field or null if either the field is empty or the given ID was <tt>null</tt>
      */
-    public <E extends BaseEntity<?>> Value lookup(Class<E> type, Object id, Mapping field) {
+    public <E extends BaseEntity<?>> Object lookup(Class<E> type, Object id, Mapping field) {
```

Comment:
why change the outside API and not simply wrap the result in a value to provide a non-null method?